Neos.Flow/Classes/Validation/Validator/RegularExpressionValidator.php

```diff
@@ -39,7 +40,11 @@ protected function isValid($value)
     {
         $result = preg_match($this->options['regularExpression'], $value);
         if ($result === 0) {
-            $this->addError('The given subject did not match the pattern. Got: %1$s', 1221565130, [$value]);
+            if ($this->options['validationErrorMessage']) {
+                $this->addError($this->options['validationErrorMessage'], 1692962252);
```

Comment:
I wonder (mostly a question to other team members with an opinion about this) do we not want to use the same error code here? After all it's the same error just the message is different.